test-apps/display-test-app/src/backend/Backend.ts

```diff
@@ -251,3 +255,65 @@ export const initializeDtaBackend = async (hostOpts?: ElectronHostOptions & Mobi
   Logger.setLevelDefault(logLevel);
   Logger.setLevel("SVT", LogLevel.Trace);
 };
+
+async function initializeAuthorizationClient(): Promise<AuthorizationClient | undefined> {
+  if (process.env.IMJS_OIDC_HEADLESS) {
+    if (checkEnvVars(
+      "IMJS_OIDC_CLIENT_ID",
+      "IMJS_OIDC_SCOPE",
+      "IMJS_OIDC_EMAIL",
+      "IMJS_OIDC_PASSWORD"
+    )) {
+      // test-only authorization client with credentials from the environment.
+      return new TestBrowserAuthorizationClient({
+        clientId: process.env.IMJS_OIDC_CLIENT_ID!,
+        redirectUri: process.env.IMJS_OIDC_REDIRECT_URI ?? "http://localhost:3000/signin-callback",
+        scope: process.env.IMJS_OIDC_SCOPE!,
+        clientSecret: process.env.IMJS_OIDC_CLIENT_SECRET,
+      }, {
+        email: process.env.IMJS_OIDC_EMAIL!,
+        password: process.env.IMJS_OIDC_PASSWORD!,
+      });
+    } else if (checkEnvVars(
+      "IMJS_OIDC_CLIENT_ID",
+      "IMJS_OIDC_CLIENT_SECRET",
+      "IMJS_OIDC_SCOPE",
+    )) {
+      // TODO: This throws a weird exception at run-time (ServiceAuthorizationClient is unable to load @itwin/core-bentley).
+      return undefined;
+      // // Setup a client using the service authorization workflow.
+      // return new ServiceAuthorizationClient({
```

Comment:
@tcobbs-bentley let's discuss this with @evelynpreslar-bentley this week and ensure we're using the best practices for the way we want to handle auth in DTA.  I don't think we want to have a headless workflow here and therefore we don't need the ServiceAuthorizationClient nor the TestBrowserAuthorizationClient.